api/types/types.go

```diff
@@ -45,6 +45,12 @@ type ImageInspect struct {
 	VirtualSize     int64
 	GraphDriver     GraphDriverData
 	RootFS          RootFS
+	Metadata        ImageMetadata
+}
+
+// ImageMetadata contains engine-local data about the image
+type ImageMetadata struct {
+	LastTagTime string
```

Comment:
should this have `omitempty`? Also wondering why it's not using `time.Time` (as we use in other places), e.g.;

    UpdatedAt time.Time `json:",omitempty"`